transformers/component_tests/queries/test_helpers/test_helpers.go

```diff
@@ -586,12 +586,15 @@ func SetUpFlopBidContext(setupData FlopBidCreationInput) (err error) {
 func CreateDeal(input DealCreationInput) (err error) {
 	addressID, addressErr := shared.GetOrCreateAddress(input.ContractAddress, input.DB)
 	Expect(addressErr).NotTo(HaveOccurred())
+	msgSenderID, msgSenderErr := shared.GetOrCreateAddress(test_data.DealEventLog.Log.Topics[1].Hex(), input.DB)
```

Comment:
Not a blocker but you could use the helper function I added for this in test_data, since we use it so often.